pallets/elections/src/lib.rs

```diff
@@ -305,6 +418,8 @@ pub mod pallet {
         ///
         /// We calculate the supports for them for the sake of eras payouts.
         fn elect() -> Result<Supports<T::AccountId>, Self::Error> {
+            Self::kick_out_underperformed_non_reserved_validators();
```

Comment:
could you add this to the comment about the order of things that happens (it is at the start of the file impls.rs)

Reply:
done